http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -112,28 +113,78 @@ class GceInterpreter[F[_]: Async: Parallel: ContextShift: Logger](
 
       initScript = GcsPath(initBucketName, GcsObjectName(config.clusterResourcesConfig.gceInitScript.asString))
 
+      bootDisk = AttachedDisk.newBuilder
+        .setBoot(true)
+        .setInitializeParams(
+          AttachedDiskInitializeParams
+            .newBuilder()
+            .setDescription("Leonardo Managed Boot Disk")
+            .setSourceImage(config.gceConfig.customGceImage.asString)
+            .setDiskSizeGb(
+              config.gceConfig.bootDiskSize.gb.toString
+            )
+            .putAllLabels(Map("leonardo" -> "true").asJava)
+            .build()
+        )
+        .setAutoDelete(true)
+        .build()
+
+      userDisk <- params.runtimeConfig match {
+        case x: RuntimeConfig.GceWithPdConfig =>
+          for {
+            diskId <- F.fromEither(
+              x.persistentDiskId
+                .toRight(new RuntimeException("Missing diskId in the request. This should never happen"))
+            )
+            persistentDiskOpt <- persistentDiskQuery.getById(diskId).transaction
+            persistentDisk <- F.fromEither(
+              persistentDiskOpt.toRight(
+                new Exception(
+                  "Runtime has Persistent Disk enabled, but we can't find it in database. This should never happen."
+                )
+              )
+            )
+          } yield AttachedDisk.newBuilder
+            .setBoot(false)
+            .setInitializeParams(
+              AttachedDiskInitializeParams
```

Comment:
Can we add a label here too as we do above?
```
.putAllLabels(Map("leonardo" -> "true").asJava)
```

Reply:
added